components/breakpoints.ts

```diff
@@ -1,6 +1,49 @@
-export const maxWidth = (breakpoint) => `max-width: ${breakpoint}px`;
-export const minWidth = (breakpoint) => `min-width: ${breakpoint + 1}px`;
+// <: extra small devices - portrait phones
+const BREAKPOINT_1 = 576;
+// > <: small devices - landscape phones
+const BREAKPOINT_2 = 768;
+// > <: medium devices - tablets
+const BREAKPOINT_3 = 992;
+// > <: large devices - desktops
+const BREAKPOINT_4 = 1200;
+// > <: extra large devices - large desktops, etc.
+const BREAKPOINT_5 = 1600;
+// >: extra extra large devices - Big McLargeHuge devices
 
-export const BREAKPOINT_DESKTOP = 1024;
-export const BREAKPOINT_IPAD = 768;
-export const BREAKPOINT_MOBILE = 576;
+function breakUpPx(breakpoint) {
+  switch (breakpoint) {
+    case "sm":
+      return BREAKPOINT_1;
+    case "md":
+      return BREAKPOINT_2;
+    case "lg":
+      return BREAKPOINT_3;
+    case "xl":
+      return BREAKPOINT_4;
+    case "xxl":
+      return BREAKPOINT_5;
+    default:
+      return breakpoint;
+  }
+}
+
+function breakDownPx(breakpoint) {
+  switch (breakpoint) {
+    case "xs":
+      return BREAKPOINT_1;
+    case "sm":
+      return BREAKPOINT_2;
+    case "md":
+      return BREAKPOINT_3;
+    case "lg":
+      return BREAKPOINT_4;
+    case "xl":
+      return BREAKPOINT_5;
+    default:
```

Comment:
No `xxl` size defined or needed here, because if you are using max-width then XXL is included by using standard CSS.